Source/content_script.js

```diff
@@ -23,7 +23,7 @@ function walk (node) {
       }
       break
 
-    case Node.TEXT_NODE
```

Comment:
LOL

Reply:
NEVER COMPROMISE
NOT EVEN IN THE FACE OF ARMAGEDDON